File: abc005/src/b.cpp
#include <algorithm>
#include <iostream>
#include <vector>

using namespace std;

int main()
{
    int n;
    cin >> n;
    vector<int> t(n);
    for (int i = 0; i < n; i++) {
        cin >> t[i];
    }
    sort(t.begin(), t.end()); // sortでascending
    int ans = t[0];           //[0]で一番小さい数字を引き出す
    cout << ans << endl;
    return 0;
}

File: abc005/src/b.go
package main

import (
	"fmt"
	"sort"
)

func main() {
	var n int
	fmt.Scan(&n)

	t := make([]int64, n)
	for i := 0; i < n; i++ {
		fmt.Scan(&t[i])
	}

	sort.Slice(t, func(i, j int) bool {
		return t[i] < t[j]
	})

	ans := t[0]
	fmt.Println(ans)
}
